### PR TITLE
Add basic structure for AddL and AddG commands

### PR DESCRIPTION
**Pull request by MadLamprey**
No functionality has been added yet, only classes have been created.

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Github.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package seedu.address.model.person;
+
+/**
+ * Represents a person's Github details in JABPro.
+ */
+public class Github {
+
+    public final String value;
```
Comment (sk2001git) [inline src/main/java/seedu/address/model/person/Github.java:10]: Should not be null , need to refer to other fields that was incorporated into person like Address, phone number.

File: src/main/java/seedu/address/model/person/Github.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    /**
+     * Constructs a {@code Github}.
+     * @param github A valid Github username
+     */
+    public Github(String github) {
+        value = github;
```
Comment (sk2001git) [inline src/main/java/seedu/address/model/person/Github.java:18]: `requireNonNull(address); `
` checkArgument(isValidAddress(address), MESSAGE_CONSTRAINTS);` - adapt accordingly, this is from address
        
   Need to have requireNonNull, and checkArgument. Might want to have a initial value as a default

File: src/main/java/seedu/address/model/person/Github.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            return true;
+        }
+
+        // instanceof handles nulls
+        if (!(other instanceof Github)) {
+            return false;
+        }
+
+        Github otherGithub = (Github) other;
+        return value.equals(otherGithub.value);
```
Comment (sk2001git) [inline src/main/java/seedu/address/model/person/Github.java:38]: Looks good, github links should be identical if they are in same in value

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 4]
```diff
@@ -21,22 +21,22 @@ public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"),
-                getTagSet("friends")),
+                getTagSet("friends"), null, null),
```
Comment (sk2001git) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java]: Please do not use null as a test value. 
Null should not be used

Comment (sk2001git) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: This applies for all test you have done for all the changes below

**Review by sk2001git — CHANGES_REQUESTED** [review]
In general just need to fix null values otherwise looks good to merge

**Comment by ScarletBlanks** [comment]
hey so one thing I noticed is that some parts don't operate under OOP standards and you're accessing public variables directly from the class     
     
        if (!linkedIn.value.isEmpty()) {
            builder.add("linkedin", linkedIn);
        }

        if (!github.value.isEmpty()) {
            builder.add("github", github);
        }

public class LinkedIn {

    public final String value;

}

public class Github {

    public final String value;

}

**Review by sk2001git — APPROVED** [review]
Test will be handed over to v1.2b 
Certain things to be addressed such as modifiers as mentioned by peng tat

otherwise looks like a good work in progress LGTM

**Pull request merged**